### PR TITLE
Readme update 2024.6

### PR DESCRIPTION
**Pull request by AlexanderBarabanov**
Readme update (2024.6)

### CODE REVIEW

**Review by ActiveChooN — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -3,34 +3,34 @@
 # OpenVINO™ Test Drive
 
 [![codecov](https://codecov.io/gh/openvinotoolkit/openvino_testdrive/graph/badge.svg?token=DH98FAPH65)](https://codecov.io/gh/openvinotoolkit/openvino_testdrive)
```
Comment (Radwan-Ibrahim) [inline README.md:5]: Let's remove this tag unit Dima fix it :)

File: README.md [hunk 2]
```diff
@@ -3,34 +3,34 @@
...
-[![openvino](https://img.shields.io/badge/openvino-2024.4-blue)]()
+[![openvino](https://img.shields.io/badge/openvino-2024.4+-blue)]()
```
Comment (Radwan-Ibrahim) [inline README.md:6]: Let's change OpenVINO to 24.6

File: README.md [hunk 3]
```diff
@@ -3,34 +3,34 @@
...
 
 </div>
 
-Get started with OpenVINO™ Test Drive, an application that allows you to run generative AI and vision models trained by [Intel® Geti™](https://docs.geti.intel.com/) directly on your computer or edge device using [OpenVINO™ Runtime](https://github.com/openvinotoolkit/openvino).
+Get started with OpenVINO™ Test Drive, an application that allows you to run generative AI and vision models trained by [Intel® Geti™](https://docs.geti.intel.com/) directly on your computer using [OpenVINO™ Runtime](https://github.com/openvinotoolkit/openvino).
```
Comment (Radwan-Ibrahim) [inline README.md:10]: Let's keep edge device. Edge device is like the NUCs. Engineers will be interested to use Test drive for testing the model performance there as well

File: README.md [hunk 4]
```diff
@@ -52,24 +52,54 @@ Upon starting the application, you can import a model using either Hugging Face
...
-3. Use `Performance metrics` tab to get model performance metrics on your computer or edge device
+3. Use `Performance metrics` tab to get LLM performance metrics on your computer.
 <p align="left">
   <img src="./docs/metrics.gif" width="500">
 </p>
 
+
+### Transcribe speech from video
+
+1. Choose a speech-to-text LLM from predefined set of popular models or pick one from Hugging Face using `Import model` -> `Hugging Face` and import it.
```
Comment (Radwan-Ibrahim) [inline README.md:73]: We have only one model Whisper :)
```suggestion
1. Try Whisper for video transcription.
```

**Review by Radwan-Ibrahim — COMMENTED** [review]
Thanks Alexander. It looks great, a few comment from side.

**Review by Radwan-Ibrahim — APPROVED** [review]
LGTM, Thanks Alexander!

**Pull request merged**